src/nos/core/notifications.py

```diff
@@ -160,6 +160,16 @@ def formatted(self, **kwargs) -> "Notification":
 """,
 )
 
+IVA_UNVERIFIED_TO_USER = Notification(
+    "Contact Address Invalidation",
+    """
+One of your contact addresses has been invalidated due to too many failed verification attempts.
+The type of contact address that was invalidated is '{type}'.
```

Comment:
Why just the type and not the value? Do you think it's a security issue to send it via email?

Reply:
My problem with the type is that it may look awkward because it will probably appear as an all uppercase enum value. Also, in 99% we will have PHONE as type anyway.

So maybe just leave it out and add the same sentence "The complete list..." as above.

Reply:
Masking the values could be difficult, because how you do that would depend on the type.

Reply:
This should happen so rarely that I don't think we need to put much effort into this.